docs/api/alerting/health.asciidoc

```diff
@@ -1,38 +1,43 @@
 [[get-alerting-framework-health-api]]
-=== Get Alerting framework health API
+== Get alerting framework health API
 ++++
-<titleabbrev>Get Alerting framework health</titleabbrev>
+<titleabbrev>Get alerting framework health</titleabbrev>
 ++++
 
-Retrieve the health status of the Alerting framework.
+Retrieve the health status of the alerting framework.
 
 [[get-alerting-framework-health-api-request]]
-==== Request
+=== {api-request-title}
 
 `GET <kibana host>:<port>/api/alerting/_health`
 
 `GET <kibana host>:<port>/s/<space_id>/api/alerting/_health`
 
+=== {api-prereq-title}
+
+You must have `read` privileges for the *Management* > *Stack Rules* feature.
```

Comment:
Users should have access to this endpoint if they have at least `read` access for any rule producer feature, not just `Stack Rules`

Reply:
Thanks for the clarification!  I've gone back and edited all the prerequisites (including the delete, update, and create rule APIs that I edited in other PRs) to make that clearer.